src/utils.ts

```diff
@@ -23,3 +23,30 @@ export function isInputLike(
     target instanceof HTMLSelectElement
   );
 }
+
+// https://github.com/grassator/canvas-text-editor/blob/master/lib/FontMetrics.js
+export function measureText(text: string, font: string) {
```

Comment:
I hoped we wouldn't need this but looks like we do. Well, at least now it's working fine :)